docs/manpage_syntax.md

```diff
@@ -0,0 +1,102 @@
+
+#### **--redhat**=**good** | _better_ | _best_
+
+This is an example for 3 (three) arguments behind an OPTION. If the number of arguments is higher than 3 (three), the arguments are **not** listed after the equal sign. The arguments have to be explained in a table like in **--test**=**_test_** regardless of the number of arguments.
+
+#### **--test**=**_test_**
+
+OPTIONS that are followed by an equal sign include an argument after the equal sign in _italic_. If there is a default argument, that is used if the OPTION is not specified in the **command**, the argument after the eqaul sign is displayed in **bold**. All arguments must be listed and explained in the text below the OPTION.
+
+| argument           | description                                                                 |
+| -                  | -                                                                           |
+| **_example one_**  | This argument is the standard argument if the OPTION is not specified.      |
+| _example two_      | If one referes to a command, one should us **bold** marks.                  |
+| _example three_    | Example: In combination with **podman command** highly effectiv.            |
+| _example four_     | Example: Can be combined with **--exit**.                                   |
+| _example five_     | The fifth description                                                       |
+
+The table shows an example for a listing of arguments. The contents in the table should be aligned left. If the content in the table conflicts with this, it can be aligned in a way that supports the understanding of the content. If there is a standard argument, it **must** listed as the first entry in the table.
+
+If the number of arguments is smaller than 4 (four) they have to be listed behind the OPTION as seen in the OPTION **--redhat**.
+
+## SUBCHAPTER
+For chapters that are made specifically as an individual SUBCHAPTER in a man page, the previous conditions regarding formatting apply.
+
+There are no restrictions for the use of paragraphs and tables. Within these paragraphs and tables the previous conditions regarding formatting apply.
+
+Strings of characters or numbers can be highlighted with `backticks`. Paths of any kind **must** be highlighted.\
+IMPORTANT: Only characters that are **not** part of categories mentioned before can be highlighted. This includes headers. For example it is not advised to highlight an OPTION or a **command**.
+
+SUBHEADINGS are displayed as follows:
+### SUBHEADING
+Text for SUBHEADINGS.
+
+## EXAMPLES
+(All EXAMPLES are listed in this section. This section should be at the end of each man page. Each EXAMPLE is always in one box. The box starts and ends with the last written line, **not** with a blank line. The `$` in front of the commands indicates that it can be run as a normal user, while the commands starting with `#` can only be run as root.)
```

Comment:
You'd two of these sections that were within parens.  I'd drop the parens as I have here and up above.  However, if you feel really strongly about them, I'd be consistent in their use throughout.

```suggestion
All EXAMPLES are listed in this section. This section should be at the end of each man page. Each EXAMPLE is always in one box. The box starts and ends with the last written line, **not** with a blank line. The `$` in front of the commands indicates that it can be run as a normal user, while the commands starting with `#` can only be run as root.
```